include/dslash_helper.cuh

```diff
@@ -4,10 +4,27 @@
 #include <gauge_field.h>
 #include <register_traits.h>
 #include <index_helper.cuh>
+#include <shmem_helper.cuh>
+
+#ifdef NVSHMEM_COMMS
+namespace
+{
+  __device__ cuda::atomic<long, cuda::thread_scope_system> intersync {0};
+  __device__ cuda::atomic<long, cuda::thread_scope_device> interior_done {0};
+  __device__ cuda::atomic<int, cuda::thread_scope_block> interior_count {0};
+} // namespace
+#endif
 
 namespace quda
 {
 
+  namespace dslash
```

Comment:
Can this be wrapped in `#ifdef NVSHMEM_COMMS`?

Reply:
THat does not seem to work for me if I open the two namespace again it still complains ...